### ISSUE TITLE
remove this bot

### ISSUE THREAD

**Issue by HexDevv** [Lachie-Source/Eval#1]
no bot list will accept it and it has so much op shit which will destroy your computer

**Comment by DefectivenessYT** [comment]
I agree, this bot will spew out it's token, and inputs are not sanitized. Please learn how to make a bot before doing so. You can also shut the bot down using a command!

**Comment by Lachie-Source** [comment]
eggy bad
